src/courtroom.cpp

```diff
@@ -2643,12 +2640,16 @@ void Courtroom::append_ic_text(QString p_text, QString p_name, QString p_action,
     else
       ui_ic_chatlog->textCursor().insertText(": ", normal);
     // Format the result according to html
-    if (log_colors)
-      ui_ic_chatlog->textCursor().insertHtml(
-          filter_ic_text(p_text, true, -1, color));
+    if (log_colors) {
+      QString p_text_filtered = filter_ic_text(p_text, true, -1, color);
+      p_text_filtered = p_text_filtered.replace("$c0", ao_app->get_color("ic_chatlog_color", "courtroom_fonts.ini").name(QColor::HexRgb));
```

Comment:
Wait this makes it so config.ini c0 will be ignored, are you sure about this

Reply:
This is done to avoid the problem of courtroom_fonts.ini's color setting being ignored entirely - it's always a possibility that users will want a different colored log than what the default c0 provides.